packages/frontend/amp/components/elements/Image.tsx

```diff
@@ -32,6 +32,12 @@ export const Image: React.FC<{
         padding-right: 3px;
     `;
 
+    injectGlobal`
+    figcaption a {
```

Comment:
I'm not sure how 'global' `injectGlobal` is but my fear would be this will target anchors in figcaptions outside of where you want it. Did adding a class not quite work?